requirements.txt

```diff
@@ -9,3 +9,4 @@ coveralls
 mpi4py
 astropy
 sphinx
+coloredlogs
```

Comment:
it would be better to discuss more largely about addition of a python package. I do not think that it is in desc-python for instance. Perhaps remove the logging addition and keep these for a future PR dedicated to this sort of things, and that can be discussed more largely?